Dockerfile

```diff
@@ -1,13 +1,10 @@
 FROM python:3.6.4-alpine3.7
 MAINTAINER tsu-shiuan@zappistore.com
```

Comment:
`MAINTAINER` is deprecated. Prefer to use `LABEL`. See:
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Something like:
```dockerfile
LABEL maintainer="SvenDowideit@home.org.au"
```